openlibrary/solr/updater/work.py

```diff
@@ -654,7 +654,11 @@ def build_identifiers(self) -> dict[str, list[str]]:
                 identifiers[k] += v
         return dict(identifiers)
 
-    def build_subjects(self) -> dict:
+    @property
+    def trending_z_score(self) -> float:
+        return self._work.get("trending_z_score", 0)
```

Comment:
I think this might not work, since `_work` is a _database_ work, not a solr work.

I think the way we might have to solve this is to make a network requests to fetch solr work data (I don't think we currently use the solr work data anywhere). We do something sort of like this in the author updater; since that has to make a network request to fetch all the author's books in solr. See if we can copy that rough pattern about when/how to make the network request. This is slightly uncharted territory, so might require a unique approach.

The way to test this is:

1. Add the book to reading logs/etc so that it's z-score gets computed and saved in solr
2. Confirm the trending score is non-zero in solr
2. Edit the the work in question ; this will trigger a new reindex of the work record


Expected: The trending score should be copied over from the previous solr record
Actual: The trending score gets zero-d by this line.